### PR TITLE
Lens v2 profile subgraph changes

### PR DESCRIPTION
**Pull request by 0xsarvesh**
- Added logic for subgraph to handle lens profile, handles, handle link, unlink & transfer
- Require module installation 

```
npx  @airstack/subgraph-generator social

```
name - lens_v2
version - 2.0.0
slug - lens_v2_polygon

### CODE REVIEW

File: lens_v2/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "test": "graph test"
   },
   "dependencies": {
-    "@airstack/subgraph-generator": "file:airstack-subgraph-generator-1.0.0-alpha.3.tgz",
+    "@airstack/subgraph-generator": "file:../airstack-modules/airstack-subgraph-generator-1.0.0-alpha.3.tgz",
```
[the remark this answers is not in the document]
Comment (0xsarvesh) [reply lens_v2/package.json:10]: I am working to get module changes published. Once its published, can change to npm

**Review by sharathkrml — CHANGES_REQUESTED** [review]

File: lens_v2/src/lens-handle.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  entity.save()
+
+  //mint is handle by Handle Minted event
+  if (event.params.from.toHexString() != ZERO_ADDRESS) {
+    airstack.social.trackSocialProfileHandleTransferTransaction(
+      event.block,
+      event.params.from.toHexString(),
+      event.params.to.toHexString(),
+      event.params.tokenId.toString(),
+      LENSHANDLE_ADDRESS.toHexString(),
```
[the remark this answers is not in the document]
Comment (0xsarvesh) [reply lens_v2/src/lens-handle.ts]: Did it whereever it was possible

File: lens_v2/src/lens-handle.ts [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    )
+  }
+// if handle is burned, make sure to un-link it.
+if(event.params.to.toHexString() == ZERO_ADDRESS) {
+
```
[the remark this answers is not in the document]
Comment (0xsarvesh) [reply lens_v2/src/lens-handle.ts]: Removed

File: lens_v2/package-lock.json
Comment (Hrishikesh-Thakkar) [inline lens_v2/package-lock.json:1]: Should we commit this?

File: lens_v2/src/lens-handle.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (sharathkrml) [reply lens_v2/src/lens-handle.ts]: ok

**Pull request merged**